Board.py

```diff
@@ -1,16 +1,134 @@
 from Card import Card
 
+
 class Board:
-    def __init__(self, cards: list()) -> None:
-        self.columns = cards
-        self.free_cells = list()
+    def __init__(self, cards: list) -> None:
+        self.columns = []
+        self.free_cells = [None for _ in range(4)]
         self.suit_stack = {'h': None, 'd': None, 'c': None, 's': None}
+        self.__make_deck(cards)
+
+    def __make_deck(self, cards: list) -> None:
+        start = 0
+        cards_per_column = [6] * 4 + [7] * 4
+
+        for num_cards in cards_per_column:
+            column = cards[start:start + num_cards]
+            self.columns.append(column)
+            start += num_cards
```

Comment:
Czy nie lepiej po prostu zrobić 
```suggestion
    def __make_deck(self, cards: list) -> None:
        current_index = 0
        while cards:   # dopóki masz nierozdane karty
            self.column[current_index].append(cards.pop())  
                       # daj kartę do kolejnej kolumny (takie klasyczne rozdawanie kart po jednej
            current_index = (current_index + 1) % 7  # zmień indeks
```
Czy coś pomijam?

Reply:
Co do dox to pewnie, zrobie to jak juz kod bedzie dzialajacy i ty bedziesz sie zajmowal freecellem.

Szczegół implementacyjny z tym rozdawaniem kart.